src/content/docs/ai-ml/ann-ai-case-study.md

```diff
@@ -0,0 +1,249 @@
+# Artifical Neural Network in use with AI
+
+| Title | Author | Tags |
+| ----- | ------ | ---- |
```

Comment:
Can you use the correct format for the header?

Here is the correct formal

```
---
title: Jenkins Case Study
author: Ernesto Osorio Jr <nyolarraklay>
tags: jenkins, case study, devOps, CI, hudson
---

```

Reply:
Fixed.